testdata/autolinks.md

```diff
@@ -0,0 +1,56 @@
+# Header https://example.org/path
+# Header https://example.org/path/
+
+Dot https://example.org/path.
+Dot https://example.org/path/.
+
+Newline https://example.org/path
+Newline https://example.org/path/
+
+Comma https://example.org/path, suffix
+Comma https://example.org/path/, suffix
+
+Semicolon https://example.org/path; suffix
+Semicolon https://example.org/path/; suffix
+
+Colon https://example.org/path: suffix
+Colon https://example.org/path/: suffix
+
+Quotes "https://example.org/path"
+Quotes "https://example.org/path/"
+
+Whitespace https://example.org/path suffix
+Whitespace https://example.org/path/ suffix
+
+https://example.org/path beginning of line
+https://example.org/path/ beginning of line
+
+https://example.org/path
+https://example.org/path/
+
+Link with parentheses https://example.org/path_(foo) and suffix.
+Link with parentheses https://example.org/path_(foo)/ and suffix.
+
+Autolink <https://example.org/path>
+Autolink <https://example.org/path/>
+
+	Code https://example.org/path
```

Comment:
The typical markdown as below will be translated.

```code
curl https://www.google.com
```

I guess this is just for lava and lava will never generate this kind of markdown.

Reply:
Exactly. That format is not used in the Lava documentation.